docs/UserGuide.md

```diff
@@ -299,6 +331,67 @@ Do you want to add another subject and grade? (yes/no)
 What else can I do for you today?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 ```
+If no student is found in the master student list, edit mode will not be entered and the programme will resume as per normal.
+
+
+#### No student found to edit:
+```
+edit
+Name of student to edit (blank to exit):
+tim lau kar meeng
```

Comment:
Once again we dont wanna use a name which might be associated with someone. Perhaps can stick with the Mario theme